microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -55,6 +68,8 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
 <!-- ====================================================================== -->
 ## Step 4 - Install the WebView2 Runtime (optional)
```

Comment:
@peiche-jessica - in this GetStart tut, delete this section "Step 4 - Install the WebView2 Runtime (optional)"?
Rendered: 
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winforms?branch=pr-en-us-2069#step-4---install-the-webview2-runtime-optional
This sibling article doesn't have such a section: 
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2069